airflow/www/static/js/dag.js

```diff
@@ -53,6 +53,7 @@ let taskId = '';
 let executionDate = '';
 let subdagId = '';
 let dagRunId = '';
+let mapIndex = undefined;
```

Comment:
```suggestion
let mapIndex = -1;
```
```suggestion
let mapIndex;
```

The linter doesn't like initializing as `undefined` so let's default to `-1` or not set anything

Reply:
I'm on this.